src/runtime/process.cpp

```diff
@@ -236,7 +236,9 @@ static obj_res spawn(string_ref const & proc_name, array_ref<string_ref> const &
 
     // Create the child process.
     bool bSuccess = CreateProcess(
-        program.c_str(),
+        // Passing `program` here should be more robust, but would require adding a `.exe` extension
+        // where necessary
+        NULL, //program.c_str(),
```

Comment:
According to the [docs](https://learn.microsoft.com/en-us/windows/win32/api/processthreadsapi/nf-processthreadsapi-createprocessa), this argument also does look in `PATH` (hence even `git.exe` would not work).

Reply:
I assume you forgot a negation there